src/Aspire.Hosting.Azure/Bicep/AzureBicepServiceBusResource.cs

```diff
@@ -29,6 +34,63 @@ public class AzureBicepServiceBusResource(string name) :
     public string? GetConnectionString() => ServiceBusEndpoint.Value;
 }
 
+/// <summary>
+/// Represents an Azure Service Bus Queue resource.
+/// </summary>
+public class AzureBicepServiceBusQueueResource(string name, AzureBicepServiceBusResource parent) :
+    Resource(name),
+    IResourceWithConnectionString,
```

Comment:
Should this really implement `IResourceWithConnectionString`? The connection string is on the namespace level.

Reply:
Nope. I don't think this needs to be a resource at all.